packer/instance-scripts/backup-chain-data.py

```diff
@@ -120,6 +122,8 @@ def backup_exists(block_number):
 # Executes the command specified in the provided argparse namespace
 def execute_command(args):
     try:
+        if args.bucket_id:
```

Comment:
nit: The style of the variable name `BACKUP_BUCKET` isn't really appropriate anymore. I would prefer we changed it to lowercase `backup_bucket` and read the value from file here, in an else block (instead of reading from file earlier and overwriting it if the arg is set.

I'd also prefer it was taken out of the `try:` block